src/Servers/HttpSys/test/FunctionalTests/AuthenticationTests.cs

```diff
@@ -368,6 +368,31 @@ public async Task AuthTypes_UnathorizedAuthenticatedAuthType_Unauthorized(Authen
             }
         }
 
+        [ConditionalTheory]
+        [InlineData(AuthenticationSchemes.Negotiate)]
+        [InlineData(AuthenticationSchemes.NTLM)]
+        // [InlineData(AuthenticationSchemes.Digest)] // TODO: Not implemented
+        // [InlineData(AuthenticationSchemes.Basic)] // Doesn't work with default creds
+        [InlineData(AuthenticationSchemes.Negotiate | AuthenticationSchemes.NTLM | /* AuthenticationSchemes.Digest |*/ AuthenticationSchemes.Basic)]
+        public async Task AuthTypes_DisableAutomaticAuthentication(AuthenticationSchemes authType)
+        {
+            using (var server = Utilities.CreateDynamicHost(out var address, options =>
+            {
+                options.AutomaticAuthentication = false;
+                options.Authentication.Schemes = authType;
+                options.Authentication.AllowAnonymous = DenyAnoymous;
+            },
+            httpContext =>
+            {
+                Assert.Null(httpContext.User);
```

Comment:
I expect this check to fail, User returns an anonymous identity rather than null.

Also call AuthenticateAsync to verify you do get the authenticated identity.

Reply:
Yeah, it's failing.
https://dev.azure.com/dnceng/public/_build/results?buildId=390076&view=ms.vss-test-web.build-test-results-tab&runId=12097012&resultId=118705&paneView=debug

Reply:
Yeah, it's failing. 
https://dev.azure.com/dnceng/public/_build/results?buildId=390076&view=ms.vss-test-web.build-test-results-tab&runId=12097012&resultId=118705&paneView=debug

Reply:
sorry I couldn't run the tests on my machine, I'm getting this error when running `restore.cmd`

> It was not possible to find any installed .NET Core SDKs

Reply:
@aspnet/build something's wrong with our bootstraping process. I hit similar issues last time I setup a machine from scratch. Any ideas?

Reply:
Ah, found it: https://github.com/aspnet/AspNetCore/blob/master/docs/BuildFromSource.md#common-error-unable-to-locate-the-net-core-sdk

Reply:
Glad you found it! Anything else needed from us here?

Reply:
No, thanks @dougbu. We'll see if that works for them.